script.sh

```diff
@@ -1,29 +1,6 @@
 #!/bin/bash
 
-mkdir -p /var/lib/node_exporter 
-chown -Rv 200:200 /var/lib/node_exporter
-
-cat  > /etc/systemd/system/smart.service << EOF
-
-[Unit]
-Description=SMART
-
-[Service]
-Type=simple
-ExecStart=/bin/sh -c "/usr/bin/smartmon.sh -r > /var/lib/node_exporter/smartmon.prom"
-SyslogIdentifier=Smartmon
-Restart=always
-RestartSec=86400
-
-[Install]
-WantedBy=multi-user.target
-
-EOF
-
-chown root:root /etc/systemd/system/smart.service
-
-chmod 644 /etc/systemd/system/smart.service
-
+systemctl daemon-reload
```

Comment:
Not a great idea to do this here, because it may inadvertently reload other daemons that the admin may not be expecting to reload yet.